core/src/test/java/io/substrait/extendedexpression/ExtendedExpressionRoundTripTest.java

```diff
@@ -0,0 +1,229 @@
+package io.substrait.extendedexpression;
+
+import io.substrait.TestBase;
+import io.substrait.expression.*;
+import io.substrait.relation.Aggregate;
+import io.substrait.relation.AggregateFunctionProtoController;
+import io.substrait.relation.ImmutableMeasure;
+import io.substrait.type.ImmutableNamedStruct;
+import io.substrait.type.Type;
+import io.substrait.type.TypeCreator;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class ExtendedExpressionRoundTripTest extends TestBase {
+  static final String NAMESPACE = "/functions_arithmetic_decimal.yaml";
```

Comment:
nit: could we put this here instead? https://github.com/substrait-io/substrait-java/blob/e73c6cac7a5b62d026ccd13eca97bfae02395aab/core/src/main/java/io/substrait/dsl/SubstraitBuilder.java#L48

Reply:
Nevermind, these are private so lets ignore this comment for now.